src/Shared/Server/infra/http.go

```diff
@@ -16,18 +16,18 @@ const (
 type HTTPServer[D, O, M any] struct {
 	router      *mux.Router
 	page        spaHandler
-	PacketRecv  chan D
-	OrderSend   chan O
-	MessageRecv chan M
+	PacketChan  chan D

```

Comment:
isnt this redundant? you already know this is a channel from the type

Reply:
Yes but I could find a better name